### PR TITLE
DEV-94: Push to Talk

### PR DESCRIPTION
**Pull request by PrestonB1123**
Changed InputDeviceButton to unmute on push and hold while in push to talk mode.
https://highfidelity.atlassian.net/browse/DEV-94

### CODE REVIEW

File: interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml [hunk 1]
```diff
@@ -77,6 +77,21 @@ Rectangle {
...
+            if (pushToTalk) {
+                AudioScriptingInterface.pushingToTalk = true;
+                Tablet.playSound(TabletEnums.ButtonClick);
+            }
+        }
+
+        onReleased: {
+            if (pushToTalk) {
+                AudioScriptingInterface.pushingToTalk = false;
+                Tablet.playSound(TabletEnums.ButtonClick);
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml]: Can you please ask Joshua if we should have a different sound associated with "button released" ?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19798/HighFidelity-Beta-PR15688-2592784.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19798/HighFidelity-Beta-PR15688-2592784.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19798/HighFidelity-Quest-Beta-PR15688-2592784.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19798/HighFidelity-Beta-PR15688-2592784.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19798/HighFidelity-Beta-Interface-PR15688-2592784.exe

**Pull request merged**